src/utilities/api-requests.ts

```diff
@@ -9,14 +63,26 @@ import axios from "axios";
  * @param defaultValue The value to return if something goes wrong
  */
 export async function request<T>(uri: string, defaultValue: T): Promise<T> {
-    let data = defaultValue;
-    await axios
-        .get(uri)
-        .then((res) => {
-            data = res.data as T;
-        })
-        .catch((error) => {
-            console.error(`Request to ${uri} failed with\n${error}`);
-        });
-    return data;
+    return attemptRequest(uri, defaultValue, 1);
```

Comment:
Codacy found an issue: ['attemptRequest' was used before it was defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5953345)